src/components/aboutUs/AboutUs.js

```diff
@@ -0,0 +1,70 @@
+import React, { useRef, useState, useEffect } from "react"
```

Comment:
the directory name itself doesn't really group together pages, really picky but a better name would be `miscellaneous`

Reply:
the folder name